context/ChainsContext/helpers.tsx

```diff
@@ -26,8 +26,7 @@ export const isChainInfoFilled = (chain: Partial<ChainInfo>): chain is ChainInfo
       chain.nodeAddresses?.length &&
       chain.denom &&
       chain.displayDenom &&
-      chain.displayDenomExponent &&
-      chain.displayDenomExponent >= 0 &&
+      Number.isSafeInteger(chain.displayDenomExponent) &&
```

Comment:
Oh, now you allowed negatives value as well. Is that intended?

Reply:
No, deleted way too many lines because the editor showed error. So I still need both the previous checks to narrow types. I'll just add the new one

Reply:
But this disables 0 haha, I need your initial check too 😯